### PR TITLE
Add statistics time window to requests

### PR DESCRIPTION
**Pull request by JamesGardiner**

### CODE REVIEW

File: rabbit_monitor.py [hunk 1]
```diff
@@ -46,13 +50,18 @@
         'messages': message_url,
         }
 
+params = {'lengths_age': settings.stats_window,
```
Comment (necrophonic) [inline rabbit_monitor.py]: Is "params" a good name? I know it's used not far below but doesn't have much context

File: settings.py [hunk 2]
```diff
@@ -14,3 +14,12 @@
 )
 
 WAIT_TIME = os.getenv('RABBIT_MONITOR_WAIT_TIME', 300)
+
+# Number of seconds to look back and gather stats from
+RABBIT_MONITOR_STATS_WINDOW = os.getenv('RABBIT_MONITOR_STATS_WINDOW',
```
Comment (necrophonic) [inline settings.py:19]: Add to readme?

Comment (JamesGardiner) [reply settings.py:19]: Yes - done :white_check_mark:

**Review by warrenbailey — APPROVED** [review]
Looks good

**Pull request merged**